src/features/import/components/Configure/Preview/index.tsx

```diff
@@ -0,0 +1,174 @@
+          <Msg id={messageIds.configuration.preview.title} />
+        </Typography>
+        <Button
+          disabled={personIndex === 0}
+          onClick={() =>
+            setPersonIndex((prev) => (personIndex !== 0 ? prev - 1 : prev))
+          }
+          startIcon={<ArrowBackIos />}
+        >
+          <Msg id={messageIds.configuration.preview.previous} />
+        </Button>
+        <Button
+          disabled={
+            personIndex ===
+            currentSheet.rows.length - (firstRowIsHeaders ? 2 : 1)
+          }
+          endIcon={<ArrowForwardIos />}
+          onClick={() =>
+            setPersonIndex((prev) =>
+              personIndex < currentSheet.rows.length - 1 ? prev + 1 : prev
+            )
+          }
+        >
+          <Msg id={messageIds.configuration.preview.next} />
+        </Button>
+      </Box>
+      <Box
+        sx={{
+          alignItems: 'center',
+          border: '1px solid lightgrey',
+          borderRadius: '5px',
+          display: 'flex',
+          height: '98px',
+          overflowX: 'auto',
+          overflowY: 'hidden',
+        }}
+      >
+        <Box
+          display="flex"
+          justifyContent="space-between"
+          sx={{ width: '100%' }}
+        >
+          {emptyPreview &&
+            Array(currentSheet.columns.length)
+              .fill(2)
+              .map((marginSize, index) => {
+                return (
+                  <Box
+                    key={`empty-preview-${index}`}
+                    flexGrow={1}
+                    sx={{
+                      backgroundColor: theme.palette.transparentGrey.light,
+                      height: '14px',
+                      m: marginSize,
+                    }}
+                  />
+                );
+              })}
+          {!emptyPreview && (
+            <>
+              {currentSheet.columns.map((column, columnIdx) => {
+                if (column.selected) {
+                  if (column.kind === ColumnKind.UNKNOWN) {
+                    return (
+                      <PreviewGrid
+                        rowValue={
+                          currentSheet.rows[
+                            firstRowIsHeaders ? personIndex + 1 : personIndex
+                          ].data[columnIdx] || (
+                            <Typography
+                              sx={{
+                                color: theme.palette.grey[400],
+                                fontStyle: 'italic',
+                              }}
+                            >
+                              (
+                              {
+                                <Msg
+                                  id={messageIds.configuration.preview.noValue}
+                                />
+                              }
+                              )
+                            </Typography>
+                          )
+                        }
+                      />
```

Comment:
This was confusing for me. I understand it now, but one reason I had to struggle to understand it at first was because it didn't have a name. That can be solved by breaking this out into a separate component and call it `EmptyPreview` or something like that. Reading a component called `EmptyPreview` makes it much easier to understand, both here where it's used, and the code inside the component.